test/service/test_request_validation.py

```diff
@@ -328,3 +328,12 @@ def test_size(self):
             with self.subTest(size=size):
                 url.args.set('size', size)
                 test(url, arg)
+
+    def test_order(self):
+        supported_orders = {'asc', 'desc'}
+        url = self.base_url.set(path='/index/projects')
+        for order in ['a', 'ascx', 'foo', 1]:
```

Comment:
Is there really a point in testing out this many invalid cases?

We should also test the valid cases.